flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -276,6 +276,11 @@ class SentryFlutterOptions extends SentryOptions {
   /// Defaults to `true`
   bool enableFramesTracking = true;
 
+  /// Maximum number of frames to track at a time before clearing.
+  /// Clearing cancels frame tracking for all active spans.
+  /// Default is 10800 frames, this equals to 3 minutes of frames at 60fps.
+  int maxFramesToTrack = 10800;
```

Comment:
l: I think a duration would be more easy to understand instead of number of frames

Reply:
the confusing part is that the callback we use from flutter doesn't actually trigger every x fps, it triggers only when new frames are drawn.

If the user doesn't do anything no scrolling etc... after 3 minutes the frames tracked might be 100 or so which doesnt align with a Duration option